README.md

```diff
@@ -106,4 +106,4 @@ If you need to support IE9, you will need to use a pollyfill for ClassList like
 Get Involved
 ---------------------
 
-Feel free to help make svelte better :-).
\ No newline at end of file
+Feel free to help make svelte better :-).
```

Comment:
Not sure why this is considered a changed line. Maybe GitHub's Web interface added a trailing newline?